### PR TITLE
doc(upgrade): improve upgrade + downgrade documentation

### PR DESCRIPTION
**Pull request by educhastenier**
following Wrap-up presentation

Closes [RUNTIME-196](https://bonitasoft.atlassian.net/browse/RUNTIME-196)

### CODE REVIEW

**Comment by pabloalonso** [comment]
Rename page to (validate we can use slash):
Bonita edition change - Community to/from Enterprise

File: modules/ROOT/pages/upgrade-from-community-to-a-subscription-edition.adoc [hunk 1]
```diff
@@ -1,59 +1,91 @@
-= Upgrade from Community to Subscription
-:description: Upgrading means moving from a Bonita Community edition environment to any Bonita Subscription
+= Upgrade from Community to Enterprise
```
Comment (pabloalonso) [inline modules/ROOT/pages/upgrade-from-community-to-a-subscription-edition.adoc]: Rename page to (validate we can use slash):
Bonita edition change - Community to/from Enterprise
```suggestion
= Bonita edition change - Community to/from Enterprise
```

Comment (educhastenier) [reply modules/ROOT/pages/upgrade-from-community-to-a-subscription-edition.adoc]: done

File: modules/ROOT/pages/upgrade-from-community-to-a-subscription-edition.adoc [hunk 2]
```diff
@@ -1,59 +1,112 @@
...
+This also applies for the downgrade.
+
+== Upgrade
 
 An upgrade is performed in two phases.
 
-== Bonita Studio upgrade
+=== Bonita studio
 
 To upgrade a Community edition Bonita Studio and its resources (such as process diagrams, data models), follow these steps:
```
Comment (pabloalonso) [inline modules/ROOT/pages/upgrade-from-community-to-a-subscription-edition.adoc]:
```suggestion
To upgrade a Community edition of Bonita Studio and its resources (such as process diagrams, data models), follow these steps:
```

**Review by pabloalonso — CHANGES_REQUESTED** [review]
Once it change is applied LGTM

File: modules/ROOT/pages/upgrade-from-community-to-a-subscription-edition.adoc [hunk 3]
```diff
@@ -1,59 +1,112 @@
...
 
-== Bonita Studio upgrade
+=== Bonita studio
 
-To upgrade a Community edition Bonita Studio and its resources (such as process diagrams, data models), follow these steps:
+To upgrade a Community edition of Bonita Studio and its resources (such as process diagrams, data models), follow these steps:
 
-. xref:bonita-bpm-studio-installation.adoc[Install a Bonita  Studio] Subscription edition.
+. xref:bonita-bpm-studio-installation.adoc[Install a Bonita  Studio] Enterprise edition.
 . Export your resources from the Bonita Studio Community edition to `.bos` files. We recommend that you keep these resource files as a backup.
```
Comment (florentsarat) [inline modules/ROOT/pages/upgrade-from-community-to-a-subscription-edition.adoc:29]: Add the git clone flow?

File: modules/ROOT/pages/upgrade-from-community-to-a-subscription-edition.adoc [hunk 4]
```diff
@@ -1,59 +1,112 @@
...
+so a prerequisite is to ensure you do not use any of these features before downgrading. Neither in your processes, nor in
+the platform settings.
+====
+
+=== Bonita studio
+
+To downgrade an Enterprise edition of Bonita Studio and its resources (such as process diagrams, data models), follow these steps:
+
+. xref:bonita-bpm-studio-installation.adoc[Install a Bonita  Studio] Community edition.
+. Export your resources from the Bonita Studio Enterprise edition to `.bos` files. We recommend that you keep these resource files as a backup.
```
Comment (florentsarat) [inline modules/ROOT/pages/upgrade-from-community-to-a-subscription-edition.adoc:92]: Add the git clone flow?

File: modules/ROOT/pages/upgrade-from-community-to-a-subscription-edition.adoc (hunk 3 above)
Comment (educhastenier) [reply modules/ROOT/pages/upgrade-from-community-to-a-subscription-edition.adoc:29]: @florentsarat What does this mean?

File: modules/ROOT/pages/upgrade-from-community-to-a-subscription-edition.adoc [hunk 5]
```diff
@@ -1,59 +1,112 @@
...
+
+The downgrade is performed automatically and silently when importing resources into Bonita Studio Enterprise edition.
+
+
+[#bonita-platform-downgrade]
+=== Bonita platform
+
+To downgrade a Bonita platform from an Enterprise edition to a Community edition, follow these steps:
+
+. Download and unzip the Bonita Community bundle (to retrieve the bonita.war Web App later)
```
Comment (dcoille) [inline modules/ROOT/pages/upgrade-from-community-to-a-subscription-edition.adoc:103]: Is there any requirements linked to the version?

File: modules/ROOT/pages/upgrade-from-community-to-a-subscription-edition.adoc [hunk 6]
```diff
@@ -1,59 +1,112 @@
...
+
+=== Bonita studio
+
+To downgrade an Enterprise edition of Bonita Studio and its resources (such as process diagrams, data models), follow these steps:
+
+. xref:bonita-bpm-studio-installation.adoc[Install a Bonita  Studio] Community edition.
+. Export your resources from the Bonita Studio Enterprise edition to `.bos` files. We recommend that you keep these resource files as a backup.
+. Import your resource files into the Bonita Studio in Community edition.
+
+The downgrade is performed automatically and silently when importing resources into Bonita Studio Enterprise edition.
```
Comment (dcoille) [inline modules/ROOT/pages/upgrade-from-community-to-a-subscription-edition.adoc]: Why precising "silently"?

Comment (educhastenier) [reply modules/ROOT/pages/upgrade-from-community-to-a-subscription-edition.adoc]: @dcoille It's the same as for the upgrade: the Studio does not issue and specific message in that case

**Review by pabloalonso — APPROVED** [review]

**Pull request merged**